scripts/linux/cleanup-k8s-and-cni.sh

```diff
@@ -33,3 +33,6 @@ iptables -t nat -F
 iptables -t mangle -F
 iptables -F
 iptables -X
+
+#reboot
+reboot
```

Comment:
I think we should use vagrant reload. Why is it so slow? Should we try debugging it?